File: ios/krk_app/BankTheme.swift
public class BankTheme: NSObject, BankThemeBase {
  
  public var color1: String = "#24207D";
  
  public var serverUrl: String = "https://online.crocusbank.ru'";
  public var udSuiteName: String = "krk";
  public var allowShowCvvCode = false
  public var showNotificationsSetting = true
  public var showBonusesPage: Bool = false
  public var statusBarTheme: String = "light"
  public var bankPhoneNumber = "+74952281244"
  public var navigationBackgroundColor: String = "#24207D"
  public var allowAddCardsToWallet = false
  public var showPersonalOffersOnMyBankPage = true;
  public var defaultMapLocation: MapLocation = MapLocation(latitude: "55.824430", longitude: "37.388902")
  public var pushNotificationsUsed: Bool = false
  public var bankMessagesUsed: Bool = true
  
  public var codeSettingPageTheme: CodeSettingPageTheme?
  
  override init() {
    self.codeSettingPageTheme = CodeSettingPageTheme(backgroundColor: self.color1,
                                       buttonBackgroundColor: "white",
                                       buttonBorderColor: self.color1,
                                       buttonTextColor: self.color1,
                                       textColor: "white",
                                       indicatorBorderColor: "white",
                                       indicatorEmptyBackgroundColor: "transparent",
                                       indicatorFullBackgroundColor: "white"
    )
  }
}

File: android/app/src/rub/java/ru/ubsmobile/BankTheme/BankTheme.java
package ru.ubsmobile.BankTheme;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactMethod;

public class BankTheme extends BankThemeBase {

    public BankTheme(ReactApplicationContext context) {
        super(context);
    }

    public static String BackgroundColor = "#0970cd";

    public void requestAutostartPermission() {
        super.requestAutostartPermission();
    }

    @Override
    public String Color1() {
        return "#0970cd";
    }

    @Override
    protected boolean AllowShowCvvCode() { return false; }

    @Override
    protected String NavigationBackgroundColor() {
        return "#0970cd";
    }

    @Override
    protected MapLocation DefaultMapLocation() {
        return new MapLocation("55.761099", "37.654990");
    }

    @Override
    protected boolean ShowBonusesPage() {
        return false;
    }

    @Override
    protected boolean ShowNotificationsSetting() {
        return false;
    }

    @Override
    protected boolean AllowAddCardsToWallet() {
        return false;
    }

    @Override
    protected boolean ShowPersonalOffersOnMyBankPage() { return false; }

    @Override
    protected CodeSettingPageTheme CodeSettingPageTheme() {
        return null;
    }

    @Override
    protected String BankPhoneNumber() {
        return "+74956483687";
    }

    @Override
    protected boolean PushNotificationsUsed() { return false; }

    @Override
    protected boolean BankMessagesUsed() { return true; }

}